linux/devdoc/file_util_linux_requirements.md

```diff
@@ -64,3 +68,64 @@ MOCKABLE_FUNCTION(, bool, file_util_close_file, HANDLE, handle_input);
 
 **SRS_FILE_UTIL_LINUX_09_019: [** `file_util_close_file` shall call `close` on the given handle_input. **]**
 
+### file_util_get_file_size_ex
+
+```c
+MOCKABLE_FUNCTION(, bool, file_util_get_file_size, PLARGE_INTEGER*, file_size, HANDLE, handle_input);
+```
+
+`file_util_get_file_size_ex` implements the GetFileSizeEx function in Linux.
```

Comment:
>in Linux. [](http://example.com/codeflow?start=67&length=9)

seems like above we mention what Linux API's we call to implement these, could be nice for these functions as well for reading at a glance.